Packages/MIES/MIES_Utilities_Algorithm.ipf

```diff
@@ -1168,3 +1168,87 @@ Function/WAVE SplitLogDataBySize(WAVE/T logData, string sep, variable lim, [vari
 
 	return result
 End
+
+Function [WAVE keySorted, WAVE dataSorted] SortKeyAndData(WAVE key, WAVE data)
+
+	ASSERT(EqualWaves(key, data, EQWAVES_DIMSIZE), "Non-matching dimension sizes for key and data")
+
+	Concatenate/FREE {key, data}, comb
```

Comment:
@t-b Does the dimension need to be specified for concatenation? e.g., /NP=1

Reply:
I could do that. But it defaults to `/NP=1` in this case as well.